week-3/Homework/mandatory/7-password-validator.js

```diff
@@ -22,48 +22,78 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
+const PreviousPassword = [
+  "fhD8!yrjj",
+  "ttkTu.wer3",
+  "dvyyeyY!5",
+  "qwbfj76%",
+  "tytT3729."
+];
+
 function validatePasswords(passwords) {
+  return passwords.map(function(password) {
+    let lenValidate = false;
+    let strnumValidate = false;
+    let symbValidate = false;
+    let IsPrevPassValidate = false;
```

Comment:
Always best to use camelCase for variable names. Rename to `isPrevPassValidate`?

https://simple.wikipedia.org/wiki/CamelCase